<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Xapient</author>
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Netzlaufwerk verbinden</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Ubuntu</family>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset theme="tag">
        <normaloff>.</normaloff>.</iconset>
      </attribute>
      <attribute name="title">
       <string>Zugangsdaten</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bitte geben sie ihre Benutzerdaten an um ihr Netzlaufwerk einzubinden!&lt;br/&gt;Ihre Daten sollten zur Sicherheit immer auf dem Netzlaufwerk abgelegt werden.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Benutzername</string>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="benutzer">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Passwort</string>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwort">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="verbinden">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0,200,70,0.4);</string>
            </property>
            <property name="text">
             <string>Verbinden</string>
            </property>
            <property name="icon">
             <iconset theme="folder-sync">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advanced">
      <attribute name="icon">
       <iconset theme="configure">
        <normaloff>.</normaloff>.</iconset>
      </attribute>
      <attribute name="title">
       <string>Einstellungen</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Bitte geben sie die IP Adresse oder der Domainnamen des Windows (samba) Servers an und definieren sie das Schema f√ºr die Freigabenamen.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton1">
            <property name="styleSheet">
             <string notr="true">margin-top:10px;</string>
            </property>
            <property name="text">
             <string>Benutzername = Freigabename</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton2">
            <property name="text">
             <string>Benutzername$ = Freigabename</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton3">
            <property name="text">
             <string>Benutzerdefinierter Freigabename</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">margin-top:6</string>
            </property>
            <property name="text">
             <string>Server</string>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="server">
            <property name="text">
             <string>10.0.0.1</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Freigabename</string>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="freigabe">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Ordner</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mountpoint">
            <property name="toolTip">
             <string>Dieser Ordner wird bei Bedarf im Homeverzeichnis erstellt</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="speichern">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0,200,70,0.4);</string>
            </property>
            <property name="text">
             <string>Speichern</string>
            </property>
            <property name="icon">
             <iconset theme="document-save">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="status">
        <property name="text">
         <string/>
        </property>
        <property name="indent">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Abbrechen</string>
        </property>
        <property name="icon">
         <iconset theme="window-close">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>benutzer</tabstop>
  <tabstop>passwort</tabstop>
  <tabstop>verbinden</tabstop>
  <tabstop>exit</tabstop>
  <tabstop>radioButton1</tabstop>
  <tabstop>radioButton2</tabstop>
  <tabstop>radioButton3</tabstop>
  <tabstop>server</tabstop>
  <tabstop>freigabe</tabstop>
  <tabstop>mountpoint</tabstop>
  <tabstop>speichern</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton2</sender>
   <signal>clicked(bool)</signal>
   <receiver>freigabe</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton1</sender>
   <signal>clicked(bool)</signal>
   <receiver>freigabe</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton3</sender>
   <signal>clicked(bool)</signal>
   <receiver>freigabe</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
